src/app/screens/Account.tsx

```diff
@@ -87,12 +102,39 @@ function AccountScreen() {
     setExportModalIsOpen(false);
   }
 
-  async function saveNostrPrivateKey(nostrPrivateKey: string) {
-    const result = await msg.request("nostr/getPrivateKey", {
-      id: account?.id,
-    });
-    const currentPrivateKey = result as unknown as string;
+  function closeNostrKeyModal() {
+    setNostrKeyModalIsOpen(false);
+  }
+
+  function generatePublicKey(priv: string) {
+    const nostr = new Nostr(priv);
```

Comment:
Should this be done via a `msg.request...` instead of importing Nostr from the background script?

Reply:
it should, we need some good folder structure. (I dislike "utils" because those are mostly bins for everything).